lib/keen/UiTextbox.vue

```diff
@@ -550,7 +601,11 @@
     transition: border 0.1s ease;
   }
 
-    .ui-textbox-textarea {
+  .ui-textbox-textarea {
```

Comment:
We need to be careful about who is rendering the border. As `.ui-textbox-label` also renders a border, and we can end up in a double-border status.

For example, look at this small diference in the border bottom on hover:
Before:
![image](https://github.com/learningequality/kolibri-design-system/assets/51239030/679496e9-2d2e-4f8a-bda3-1bb77ffd42ff)

After:
![image](https://github.com/learningequality/kolibri-design-system/assets/51239030/64da5ac1-4e24-4673-bb17-55c67905861b)


And when `invalid` and `invalidText` are set:

Before:
![image](https://github.com/learningequality/kolibri-design-system/assets/51239030/9dde9c56-148b-4e28-a18a-ab96f60cdf66)


After:
![image](https://github.com/learningequality/kolibri-design-system/assets/51239030/cb8c0f39-20ee-4d2e-aea2-c3b5836cdb2d)

This is a small diference because `.ui-input-content` is now rendering a border that is probably not necessary

Reply:
I believe this is now working as expected :) 

![a really long invalid text ](https://github.com/learningequality/kolibri-design-system/assets/59121896/d53cc5ff-44a5-48c3-bc9f-cff8c6354bfd)